.github/workflows/review.yml

```diff
@@ -30,6 +30,9 @@ jobs:
           DOCKER_BUILD_ARG_NEXT_PUBLIC_SENTRY_DSN: ${{ secrets.GH_SENTRY_DSN }}
           DOCKER_BUILD_ARG_NEXT_PUBLIC_ENVIRONMENT: 'review'
           DOCKER_BUILD_ARG_NEXT_PUBLIC_API_BASE_URL: 'https://palvelutarjotin-api.test.kuva.hel.ninja/graphql'
+          DOCKER_BUILD_ARG_NEXT_PUBLIC_MATOMO_URL_BASE: 'https://analytics.hel.ninja/'
```

Comment:
does this pollute production analytics with test env analytics? Test environments should have it's own url?

Reply:
It is not activated in staging for example. Maybe we could delete the env variables from envs where they are not really used

Reply:
So this can be / should be removed + also from staging.yml ?